integration-tests/tests/service_test.go

```diff
@@ -1,112 +0,0 @@
-// -*- Mode: Go; indent-tabs-mode: t -*-
-// +build !excludeintegration,!excludereboots
-
-/*
- * Copyright (C) 2015, 2016 Canonical Ltd
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 3 as
- * published by the Free Software Foundation.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- *
- */
-
-package tests
-
-import (
-	"fmt"
-	"os"
-	"regexp"
-
-	"github.com/ubuntu-core/snappy/integration-tests/testutils/build"
-	"github.com/ubuntu-core/snappy/integration-tests/testutils/cli"
-	"github.com/ubuntu-core/snappy/integration-tests/testutils/common"
-	"github.com/ubuntu-core/snappy/integration-tests/testutils/data"
-	"github.com/ubuntu-core/snappy/integration-tests/testutils/updates"
-	"github.com/ubuntu-core/snappy/integration-tests/testutils/wait"
-	"github.com/ubuntu-core/snappy/testutil"
-
-	"gopkg.in/check.v1"
-)
-
-var _ = check.Suite(&serviceSuite{})
-
-type serviceSuite struct {
-	common.SnappySuite
-}
-
-func (s *serviceSuite) SetUpTest(c *check.C) {
-	s.SnappySuite.SetUpTest(c)
-
-	c.Skip("FIXME: we have no snap build nor snapcraft build yet")
```

Comment:
I think we can keep (and reactivate) this one and snapBuildSuite, if they are skipped only because build.localSnap it is currently working https://github.com/ubuntu-core/snappy/blob/master/integration-tests/tests/interfaces_test.go#L61